### PR TITLE
Bare metal forms and helpers

### PR DESCRIPTION
**Pull request by alfredoC10**
Hello software engineers, Thanks for the time you take to review the repository!
The goal consisted of building a sign-in form and an edit user form that adds users's information to a database. First I build the forms view almost all in HTML and next with the Rails helpers.

Features used.
- HTML form tag
- form_tag tag
- form_for tag
-  label_tag
- password_field_tag
- f.text_field tag
-  Procedure for setting up  a basic Rails app

Have a good day!

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__You've made laudable progress in this project.
However, some changes are required before it can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] I could not access your project. After cloning and all other necessary set-ups, I tried to access `localhost:3000`, and below the error page:
error1

- [x] It’s better to delete **commented piece of codes**, as you have in `app/controllers/users_controller.rb`. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  

- [ ] You did not have linters and GitHub Actions set up in your project. Kindly set up your [linters](https://github.com/microverseinc/linters-config/tree/master/ror) and ensure that all checks are passed before requesting another review.



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Comment by alfredoC10** [comment]
Thanks for the timely observations. So these issues don't cause future malfunctions any more, I will do it all in my hands .

**Review by Temmarie — CHANGES_REQUESTED** [review]
## Changes Required! :recycle:

### Hi @alfredoC10   ! :wave: :wave:

#### You have done well so far and your project works perfectly fine. :clap: However, there are changes required for you to make in order for this project to be approved. Kindly work on them and submit them for another code review.


**Recommended**

- [ ] Please fix your rubocop errors. Most of them can be fixed by running `rubocop -a` in your terminal.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Comment by alfredoC10** [comment]
Thank you @Temmarie for the keen eye, I hadn't noticed these rubocop errors !

**Review by Georjane — CHANGES_REQUESTED** [review]
## Status: Changes Required 🔁🚫 
### Hi Team 🙌

Nice project you got there but your linters are still not passing. Please fix your rubocob errors and make sure your linters are passing. Then you can submit for another code review. 

Please feel free to reach out to me on Slack if you have any questions, *Witah Georjane* 🤝
Goodluck on your project and Happy Coding 💻

**Comment by alfredoC10** [comment]
Good day miss Georjane, i feel lucky that my code has been reviewed by such a proactive software developer, for sure very productive in her duty! and I find the review very reasonable:  by fixing the rubocop errors in previous projects I have seen how the application started working. 
What I see now is that, in my end, when I fix a rubocop error and do git push to Github, I'm  still receiving the same 39 Rubocop offenses. This makes me think that these errors might come directly from the rails default configuration so there's nothing I can do. 
But Im sure I'm not seeing the whole picture, so I´m open to any information you consider essential I must know. 
Have a nice day, I'm available in slack for anything related.

File: app/helpers/application_helper.rb
Comment (teekaytech) [inline app/helpers/application_helper.rb]: Enabling/disabling these cops might not be necessary

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__You've done well so far: everything seems to be working :clap: 
However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] Please note that all GitHub Action checks must be passed before any project can be approved. At the moment, you're still having failed checks. Here's a suggestion of what to do:
- You can remove the `# frozen_string_literal: true` from the first line of all the affected files: you did not actually need that statement on those files.
- If you're still not sure of what the problem is, or you're stuck at any point, feel free to [ask for help from a  full-time TSE](https://microverse.zendesk.com/hc/en-us/articles/360026625973-How-can-I-get-help-from-TSEs-).



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

File: app/helpers/application_helper.rb
Comment (alfredoC10) [reply app/helpers/application_helper.rb]: This more detailed description of the guidelines is very useful to me, thank you Mr @teekaytech, now I see better how to align myself to the project requirements, and also I'm going to take your word and address to a full time TSE because when I remove a # frozen_string_literal: true from a file and run rubocop next in the terminal, it says that a # frozen_string_literal: true is missing in that file, so I´m still confused with this problem. Here is a screen shot:

Comment (alfredoC10) [reply app/helpers/application_helper.rb]: image

**Review by SmithVyne — APPROVED** [review]
Project Approved 👋 🏅 
Hello @alfredoC10, congratulations, you implemented all the required changes requested by the previous TSEs and your project is approved.

You can now merge this branch into development and proceed further in the technical curriculum. Reach out to me on Slack, **smithvyne**, if you have any further questions.

**Pull request merged**
